config/foxglove-sim.json

```diff
@@ -0,0 +1,257 @@
+        },
+        "frame:link5": {
+          "visible": false
+        },
+        "frame:link2": {
+          "visible": false
+        },
+        "frame:link3": {
+          "visible": false
+        },
+        "frame:link4": {
+          "visible": false
+        },
+        "frame:gripper_left_link": {
+          "visible": false
+        },
+        "frame:gripper_right_link": {
+          "visible": false
+        }
+      },
+      "topics": {
+        "/ouster/scan": {
+          "visible": false,
+          "colorField": "intensity",
+          "colorMode": "colormap",
+          "colorMap": "turbo"
+        },
+        "/ouster/points": {
+          "visible": true,
+          "colorField": "intensity",
+          "colorMode": "colormap",
+          "colorMap": "rainbow"
+        },
+        "/oak/rgb/image_raw": {
+          "visible": false
+        },
+        "/panther/lights/driver/front_panel_frame": {
+          "visible": false
+        },
+        "/map": {
+          "visible": true
+        },
+        "/scan_filtered": {
+          "visible": true,
+          "colorField": "range",
+          "colorMode": "colormap",
+          "colorMap": "rainbow",
+          "pointSize": 6
+        },
+        "/local_costmap/costmap": {
+          "visible": true,
+          "frameLocked": false,
+          "minColor": "#5f00bd47",
+          "maxColor": "#af000059",
+          "unknownColor": "#80808082",
+          "invalidColor": "#ff0000"
+        },
+        "/local_costmap/clearing_endpoints": {
+          "visible": false,
+          "colorField": "x",
+          "colorMode": "colormap",
+          "colorMap": "turbo"
+        },
+        "/global_costmap/costmap": {
+          "visible": true,
+          "minColor": "#5f00bd47",
+          "maxColor": "#af000059",
+          "unknownColor": "#80808082",
+          "invalidColor": "#ff0000ff"
+        },
+        "/plan": {
+          "visible": false
+        },
+        "/local_plan": {
+          "visible": true,
+          "lineWidth": 0.02,
+          "gradient": [
+            "#6bff76",
+            "#77ff6b"
+          ],
+          "type": "line"
+        },
+        "/plan_smoothed": {
+          "visible": false
+        },
+        "/move_base_simple/goal": {
+          "visible": true
+        },
+        "/received_global_plan": {
+          "visible": true,
+          "lineWidth": 0.02,
+          "gradient": [
+            "#fd0000",
+            "#ff6b6b80"
+          ],
+          "type": "line"
+        },
+        "/transformed_global_plan": {
+          "visible": false
+        }
+      },
+      "layers": {
+        "b1115faf-0465-4daa-9b14-902aa3c8f7ad": {
+          "visible": true,
+          "frameLocked": true,
+          "label": "URDF",
+          "instanceId": "b1115faf-0465-4daa-9b14-902aa3c8f7ad",
+          "layerId": "foxglove.Urdf",
+          "url": "http://localhost:8080/rosbot_xl_manipulation.urdf",
+          "order": 1
+        }
```

Comment:
Have you checked if it is needed?
If the urdf is identical to that in [rosbot_xl_manipulation](https://github.com/husarion/rosbot_xl_manipulation_ros/tree/main/rosbot_xl_manipulation_description/urdf), it should be enough to listen to the `/robot_description` topic instead of looking directly to the file. In this case, adding a volume and adding a custom urdf layout is not necessary. 

Unless there is a risk of having to change the urdf in the future, you can leave it as it is.

Reply:
The `/open_manipulator_x_description` is not installed

Reply:
Foxglove can't access the open_x_manipulator meshes

Reply:
Your right, I add this to foxglovebridge-docker [PR](https://github.com/husarion/foxglove-bridge-docker/pull/4), but I can stay as it is because I believe you do not have time to test all of this.